tipCalculator.js

```diff
@@ -0,0 +1,38 @@
+$('.plus').click(function(){
+    $('#people').val(Number($('#people').val()) + 1)   ;
+});
+
+$('.minus').click(function(){
+    if($('#people').val() == '1'){
+        $('#people').val(1);
+    } else {
+        $('#people').val(Number($('#people').val()) - 1)   ;
+    }
+});
+
+
+$('.submit').click(function(e){
+    e.preventDefault();
+    var bill     = Number($('#bill').val());
+    var people   = Number($('#people').val());
+    var quality  = Number($("input[name='quality']:checked").val()) /100;
+
+    if(!isNaN(quality)){
+        if(bill < 0 || people <= 0){
```

Comment:
Here if `bill` is 0, then it will also be falsy, so you can get rid of the `< 0`. I'm not sure the situation where `people` would be less than zero. Is the `=` a typo? If so, you can change this line to `if(bill || people) {`